src/cdk/drag-drop/directives/drag.ts

```diff
@@ -139,6 +141,18 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
   /** Class to be added to the preview element. */
   @Input('cdkDragPreviewClass') previewClass: string | string[];
 
+  /**
+   *  Target for Drag Preview clone to append into.
+   *  `global` will place the clone at the end of `<body>`.
+   *  `parent` will place the clone in the parent to the `CdkDropList` .
+   *  `ElementRef` or `HTMLElement` will place the clone inside the element.
+   *
+   *  Setting this to anything other than `global` may impact how `z-index` and `overflow`
+   *  are computed.
+   */
+  @Input('cdkDragPreviewContainer') previewContainer:
+    'global' | 'parent' |ElementRef|HTMLElement;
```

Comment:
The spacing between elements here is inconsistent.

Reply:
ooh good catch :-)